CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Castle.Facilities.Quartz Changelog
+
+## 0.5.0
+- Compatible with .NET Standard 2.0
+- BREAKING CHANGE: namespace is changed into Castle.Facilities.Quartz. Code remains the same
+
+
+## 0.4.0
```

Comment:
Is there a reason you've got 0.4.0 and 0.5.0 when neither have been released? I just saw the csproj has 0.5.0, probably should pick a single version for the next release.

Reply:
@jonorossi 0.5.0 is next version